### PR TITLE
Adds go.scottylabs.org config

### LINKED ISSUE #20: Integrate with go.scottylabs.org/

### PR DESCRIPTION
**Pull request by anudaweerasinghe**
Resolves #20 

@hdavidethan we should probably move go.scottylabs to scotty.lol/ or go.scotty.lol/

### CODE REVIEW

File: commands/scottylabs.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: ScottyLabs
```
Comment (hdavidethan) [inline commands/scottylabs.yml]: ScottyLabs Go

Comment (hdavidethan) [reply commands/scottylabs.yml]: nvm I see the homepage

Comment (anudaweerasinghe) [reply commands/scottylabs.yml]: rip i already changed it

**Review by hdavidethan — APPROVED** [review]
[LGTM](https://lgtm.lol/i/759)

**Pull request merged**